Backend/src/main/java/idealab/api/model/PrintJob.java

```diff
@@ -30,9 +30,6 @@
     @Enumerated(EnumType.STRING)
     private Status status;
 
-    @Column(name = "employee_notes",  nullable = false)
-    private String employeeNotes;
-
     @OneToOne(targetEntity=Queue.class, mappedBy="printJobId")
     private Queue queueId;
```

Comment:
I know you didnt change anything here but I think it's better name this variable as queue. Since it's not really id.

Reply:
@kaansonmezoz Great idea!

Since it will be a decent size refactor since we have multiple foreign keys that say Id in them, I created an issue here: https://github.com/codefordenver/ideaLab/issues/112

Reply:
Yeah you are right, it's a better to create an issue for refactoring since it's going to take place in many source codes :) So I'm going to change my review into approved :)

Reply:
@kaansonmezoz  Thank you! And thanks for such quick turnaround with the pull requests :)